src/modules/processing/servicesPatternTesters.js

```diff
@@ -58,4 +58,7 @@ export const testers = {
 
     "youtube": (patternMatch) =>
         patternMatch.id?.length <= 11,
+
+    "newgrounds": (patternMatch) =>
+        patternMatch.id?.length <= 6,
```

Comment:
https://www.newgrounds.com/audio/listen/1 exists
```suggestion
        patternMatch.id?.length <= 1,
```